app/mod_auth/controllers.py

```diff
@@ -0,0 +1,193 @@
+mod_auth = Blueprint('auth', __name__, url_prefix='/regions')
+
+
+# Set the route and accepted methods
+@mod_auth.route('/<regionid>', methods=['GET'])
+@authorized
+@check_region
+def get_status(regionid, token=None):
+    """
+    Lists information the status of the synchronization of the images in
+    the region <regionid>. Keep in mind that <regionid> is the name of
+    the region.
+    :param regionid: Region name how you can obtain from the Keystone
+                     service. Example: Spain2.
+    :return: JSON response message with the detailed information about
+             the images and the sincronization status.
+    """
+
+    message = "GET, get information about the synchronization status in the region: {}".format(regionid)
+
+    logger.info(message)
+
+    # Just for check this data should be returned by glancesync client
+    x = Images()
+
+    value = ['3cfeaf3f0103b9637bb3fcfe691fce1e', 'base_ubuntu_14.04', 'ok', None]
+    x.add(value)
+
+    value = ['4rds4f3f0103b9637bb3fcfe691fce1e', 'base_centOS_7', 'ok', None]
+    x.add(value)
+
+    resp = make_response(x.dump(), httplib.OK)
+    resp.headers[SERVER_HEADER] = SERVER
+    resp.headers[CONTENT_TYPE] = JSON_TYPE
+
+    return resp
+
+
+@mod_auth.route('/<regionid>', methods=['POST'])
+@authorized
+@check_region
+def synchronize(regionid, token=None):
+    """
+    Synchronize the images of the corresponding region defined by its regionId.
+    The operation is asynchronous a response a taskId in order that you can follow
+    the execution of the process.
+    :param regionid: Region name how you can obtain from the Keystone
+                     service. Example: Spain2.
+    :return: JSON message with the identification of the created task
+    """
+    # WARNING
+    # It is for testing only, the functionality of this method is create a new Task,
+    # store the content of the new task with <taskid> in DB with status 'syncing'
+    # and launch a fork to start the execution of the synchronization process. At the
+    # end of the synchronization process, we update the DB registry with the status
+    # 'synced'
+    message = "POST, create a new synchronization task in the region: {}".format(regionid)
+
+    logger.info(message)
+
+    newtask = Task(status='syncing')
+
+    # name and role should be returned from authorized operation, to be extended in Sprint 16.02
+    newuser = User(region=regionid, name=token.username, taskid=str(newtask.taskid),
+                   role='admin', status=newtask.status)
+
+    db.session.add(newuser)
+    db.session.commit()
+
+    resp = make_response(newtask.dump(), httplib.OK)
+    resp.headers[SERVER_HEADER] = SERVER
+    resp.headers[CONTENT_TYPE] = JSON_TYPE
+
+    return resp
+
+
+@mod_auth.route('/<regionid>/tasks/<taskid>', methods=['GET'])
+@authorized
+@check_region
+def get_task(regionid, taskid, token=None):
+    """
+
+    :param regionid:
+    :param taskid:
+    :return:
+    """
```

Comment:
This documentation with blank descriptions does not provide helpful additional information...